platform/mbed_retarget.cpp

```diff
@@ -904,21 +904,71 @@ extern "C" long PREFIX(_flen)(FILEHANDLE fh)
     return size;
 }
 
-extern "C" char Image$$RW_IRAM1$$ZI$$Limit[];
+#if defined(TARGET_NUVOTON) || defined(TARGET_RZ_A1XX)
+extern char Image$$ARM_LIB_HEAP$$Base[];
+extern char Image$$ARM_LIB_STACK$$Base[];
+extern char Image$$ARM_LIB_HEAP$$Limit[];
```

Comment:
According to [Image$$ execution region symbols](http://www.keil.com/support/man/docs/armlink/armlink_pge1362065952432.htm), `Image$$ARM_LIB_HEAP$$Limit` doesn't include ZI section. Change to `Image$$ARM_LIB_HEAP$$ZI$$Limit`?

Reply:
Fixed.